test/switch_test.js

```diff
@@ -161,9 +210,8 @@ describe.skip('switch', () => {
           case c:
             return d;
         }
-      };
-
-      e;
+      }
+      ;e;
```

Comment:
This is due to the way CoffeeScript encodes location information about functions. Oddly, it does the same thing CoffeeScriptRedux does where they consume as much non-semantic source as possible after the end of the function and include it in the function's range, so the placement of the semicolons afterward is wrong. Don't worry about this, it needs to be fixed in decaffeinate-parser.